### PR TITLE
Join the multi01 cluster to the CI build farm

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/test-platform @hongkailiu 
[DPTP-3353](https://issues.redhat.com//browse/DPTP-3353)

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

**Comment by hongkailiu** [comment]
I think you removed one change which caused the failure of `ci/prow/secret-generator-config-valid`.

/hold

Let us do the config-updater's kubeconfig first.

**Comment by droslean** [comment]
@hongkailiu After some investigation, I realized what was missing and why the job failed. I created all the entries in Vault and updated the PR. Please check and let me know if anything else is missing.

**Comment by droslean** [comment]
@hongkailiu The PR seems to be ready for review. I have added the `config-updater` kubeconfig in https://github.com/openshift/release/pull/38446 and the rehearsal now is passing.

File: clusters/build-clusters/_cluster-init.yaml [hunk 1]
```diff
@@ -4,3 +4,4 @@ managed:
 - build03
 - build04
 - build05
+- multi01
```
Comment (hongkailiu) [inline clusters/build-clusters/_cluster-init.yaml:7]: nip: not sure if this reflects the fact. ^_^

**Review by hongkailiu — COMMENTED** [review]
/lgtm

**Pull request merged**